src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
-export interface Figure {
+export enum Shapes {
+  triangle = 'triangle',
+  circle ='circle',
+  rectangle = 'rectangle'
+}
 
+export enum Colors {
+  'red',
+  'green',
+  'blue',
 }
```

Comment:
Приведи enum в единый формат
Качество кода важно для трудоустройства
```suggestion
export enum Shapes {
  Triangle = 'triangle',
  Circle = 'circle',
  Rectangle = 'rectangle'
}

export enum Colors {
  Red = 'red',
  Green = 'green',
  Blue = 'blue',
}
```